app/src/main/java/fr/free/nrw/commons/contributions/ContributionsListFragment.java

```diff
@@ -156,6 +158,29 @@ public void onItemRangeInserted(int positionStart, int itemCount) {
         }
       }
     });
+
+    //Fab close on touch outside (Scrolling or taping on item triggers this action).
+    rvContributionsList.addOnItemTouchListener(new OnItemTouchListener() {
+      @Override
+      public boolean onInterceptTouchEvent(@NonNull RecyclerView rv, @NonNull MotionEvent e) {
+        if (e.getAction() == MotionEvent.ACTION_DOWN) {
+          if (isFabOpen) {
+            animateFAB(isFabOpen);
+          }
+        }
+        return false;
+      }
+
+      @Override
```

Comment:
Ah, sorry, would you mind adding a javadoc to these two methods just saying why it is important that these methods are written here but kept empty? Otherwise someone might want to remove them.
As a rule, we add javadoc for all new written methods/etc, saying what's the purpose of having them.
Thanks!

Reply:
Added official javadoc 
I forget those everytime : / 
Thanks.